src/css/controls/imports/tooltip.less

```diff
@@ -4,10 +4,6 @@
     opacity: 1;
     transition-delay: 0s;
     visibility: visible;
-
-    &:hover {
-        outline: 0;
```

Comment:
why are we removing this ? tooltips should not have an outiline when opened

Reply:
This is for the volume slider. The text (simple) tooltips don't have the `.jw-overlay` class. This line was actually added by me in a previous PR.